packages/daemon/src/config/config.default.ts

```diff
@@ -30,5 +30,8 @@ export default (appInfo: EggAppInfo) => {
     dir: staticDir,
   };
 
+  // sequelize logger: boolean or function
+  config.sequelizeLogger = false;
```

Comment:
How about providing an environment variable to config this.

Reply:
As a daemon process, config file is more explicit and has a more appropriate scope.